packages/mintlify-cli/src/index.ts

```diff
@@ -21,6 +25,25 @@ export const cli = () =>
       () => {},
       installDepsCommand
     )
+    .command(
+      "broken-links",
+      "Check for broken links in your Mintlify project",
+      () => {},
+      async () => {
+        await getBrokenInternalLinks(".");
+      }
+    )
+    .command(
+      "rename",
+      "Rename a file and update links in content",
+      () => {},
+      async (argv) => {
+        await renameFileAndUpdateLinksInContent(
+          argv.from as string,
+          argv.to as string
+        );
+      }
```

Comment:
I think this can be shortened as well: 
```suggestion
     (argv) =>
      renameFileAndUpdateLinksInContent(
        argv.from as string,
        argv.to as string
      )
```